src/cache/cache_las.c

```diff
@@ -1423,10 +1434,50 @@ __wt_find_lookaside_upd(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt, WT_UPDAT
         upd->durable_ts = durable_timestamp;
         upd->start_ts = las_timestamp;
         upd->prepare_state = prepare_state;
-        /*
-         * Mark this update as external and to be discarded when not needed.
-         */
-        upd->ext = 1;
+
+        if (prepare_state == WT_PREPARE_INPROGRESS) {
```

Comment:
Can you put a comment regarding what this block of code is expected to do

Reply:
fd82c8e

The relevant comment was above the `remove` call. I've moved most of that comment to above the block.